src/lib/seam/components/AccessCodeForm/TimezonePicker/TimezonePicker.tsx

```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames'
+import { useState } from 'react'
+
+import {
+  getBrowserTimezone,
+  getTimezoneOffsetMinutes,
+  getTimezones,
+} from 'lib/dates.js'
+import { TimezoneListItem } from 'lib/seam/components/AccessCodeForm/TimezonePicker/TimezonePickerListItem.js'
+import { Checkbox } from 'lib/ui/Checkbox.js'
+import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
+
+interface TimezonePickerProps {
+  value: string
+  onChange: (timezone: string) => void
+  onClose: () => void
+}
+
+export function TimezonePicker({
+  onChange,
+  value,
+  onClose,
+}: TimezonePickerProps): JSX.Element {
+  const [manualTimezoneEnabled, setManualTimezoneEnabled] = useState(false)
+
+  const isBrowserTimezone = value === getBrowserTimezone()
+  const isManualTimezone = !isBrowserTimezone || manualTimezoneEnabled
+
+  const handleChangeManualTimezone = (enabled: boolean): void => {
+    setManualTimezoneEnabled(enabled)
+    if (!enabled) {
+      onChange(getBrowserTimezone())
+    }
+  }
+
+  const title = isManualTimezone ? t.titleManual : t.titleAuto
+
+  return (
+    <div className='seam-timezone-picker'>
+      <ContentHeader title={title} onBack={onClose} />
+      <div className='seam-content'>
+        <Checkbox
+          label={t.setTimezoneManuallyLabel}
+          checked={isManualTimezone}
+          onChange={handleChangeManualTimezone}
+          className='seam-manual-timezone-checkbox'
+        />
+        <div
+          className={classNames('seam-timezones', {
+            'seam-disabled': !isManualTimezone,
+          })}
+        >
+          <ul>
+            {getTimezones()
+              .sort(
```

Comment:
Let's move this to [dates.ts](https://github.com/seamapi/react/blob/main/src/lib/dates.ts) as `compareByTimezoneOffsetAsc`

Reply:
Added in https://github.com/seamapi/react/pull/282/commits/3f987acb610ff1e28b52c96caacc4b18140d0935